### PR TITLE
Tokenomics and builders fixes

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

**Review by bgodlin — APPROVED** [review]

File: .vitepress/config.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export default defineConfig({
               { text: "Fearless Wallet", link: "fearless" },
             ],
           },
-          { text: "Request Features", link: "rfp" },
+	 { text: "Request Features", link: "rfp" }
```
Comment (outoftardis) [inline .vitepress/config.ts]: revert this change please

File: src/sora-builders.md
Comment (outoftardis) [inline src/sora-builders.md:1]: in the build.md file there is a link to the decentralised-development-process.md

line 68: https://github.com/sora-xor/sora-docs/blob/develop/src/build.md?plain=1#L68

File: src/tokenomics.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
 # Tokenomics
 
-<!-- TODO: Add intro? -->
+In this section you will find all the information on SORA
+tokenomics. Starting with the Token Bonding curve, each subsection
+will cover the main [SORA economy](sora-economy.md) tokens. It is
+important to remember that the [XOR](xor.md) token does not follow
+conventional tokenomics of limited supply, instead, it's supply is
```
Comment (outoftardis) [inline src/tokenomics.md]:
```suggestion
conventional tokenomics of limited supply, instead, its supply is
```

😆

File: src/tokenomics.md [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
+- [XOR](xor.md) is the network utility token used for transaction fees (gas)
+- [VAL](val.md) is the token awarded to validators for securing the
+  SORA network
+- [PSWAP](pswap.md) is the LP reward token for liquidity providers on
+  [Polkaswap](https://polkaswap.io/) and is burnt with every transaction.
+- [XST](xst.md) is the underlying asset for the creation of SORA
+  Synthetics
+- The [Token Bonding Curve Dollar (TBCD)](tbcd.md) is an algorithmic,
+  non-synthetic stablecoin that has its value maintained by the SORA
+  token bonding curve.
```
Comment (outoftardis) [inline src/tokenomics.md]:
```suggestion
- The [Token Bonding Curve (TBC)](tbc.md) is the smart contract that
  manages the supply of XOR in a rational way, without the involvment
  of humans.
- [XOR](xor.md) is the network utility token used for transaction fees (gas).
- [VAL](val.md) is the token awarded to validators for securing the
  SORA network.
- [PSWAP](pswap.md) is the LP reward token for liquidity providers on
  [Polkaswap](https://polkaswap.io/) and is burnt with every transaction.
- [XST](xst.md) is the underlying asset for the creation of SORA
  Synthetics.
- The [Token Bonding Curve Dollar (TBCD)](tbcd.md) is an algorithmic,
  non-synthetic stablecoin that has its value maintained by the SORA
  token bonding curve.
```

File: src/changelog.md [hunk 4]
```diff
@@ -1,15 +1,31 @@
-[SORA mainnet](https://github.com/sora-xor/sora2-network/releases)
+# SORA Network Development Changelog
 
-[SORA parachain](https://github.com/sora-xor/sora2-parachain/releases)
+The SORA network and SORA builders are contantly working to improve
```
Comment (outoftardis) [inline src/changelog.md:3]:
```suggestion
The SORA network and SORA builders are constantly working to improve
```

File: src/changelog.md [hunk 5]
```diff
@@ -1,15 +1,31 @@
...
+the network and add more features and stability. In this page you will
```
Comment (outoftardis) [inline src/changelog.md:4]:
```suggestion
the network and add more features and stability. On this page you will
```

File: src/changelog.md [hunk 6]
```diff
@@ -1,15 +1,31 @@
...
+- [SORA parachain](https://github.com/sora-xor/sora2-parachain/releases)
 
-[iOS]()
+## SORA Mobile
 
-[Android]()
+- [iOS]()
 
-**_TODO_** How can it be combined with the regular ecosystem updates?
+- [Android]()
```
Comment (outoftardis) [inline src/changelog.md]: btw these links, if left like this, will point to the sora-docs root page

File: src/changelog.md [hunk 7]
```diff
@@ -1,15 +1,31 @@
...
+
+- [Polkaswap](https://github.com/sora-xor/polkaswap-exchange-web/releases)
+
+## SORA Builders
+
+- [ADAR]()
+
+- [Demeter]()
+
+- [Fearless Wallet]()
```
Comment (outoftardis) [inline src/changelog.md]: same comment about links as above

File: .vitepress/config.ts (hunk 1 above)
Comment (outoftardis) [reply .vitepress/config.ts]:
```suggestion
          { text: "Request Features", link: "rfp" },
```

File: src/tbcd.md [hunk 8]
```diff
@@ -1,3 +1,64 @@
 # Token Bonding Curve Dollar (TBCD)
 
-<!-- TODO: Add info -->
+- Token Bonding Curve Dollars (TBCD) is a new proposed, algorithmic, non-synthetic stablecoin that has its value maintained by the SORA token bonding curve
+- TBCD is created to be worth slightly less than $1 when XOR is far lower than the price point on the token bonding curve, reducing sell pressure to XOR when the price of XOR is low
+- TBCD will help to build up reserves in the SORA token bonding curve, while also being used to fund the creation of new goods and services via on-chain referendum decided by all XOR token holders
```
Comment (outoftardis) [inline src/tbcd.md:5]:
```suggestion
- Token Bonding Curve Dollars (TBCD) is a new proposed, algorithmic, non-synthetic stablecoin that has its value maintained by the SORA token bonding curve.
- TBCD is created to be worth slightly less than $1 when XOR is far lower than the price point on the token bonding curve, reducing sell pressure to XOR when the price of XOR is low.
- TBCD will help to build up reserves in the SORA token bonding curve, while also being used to fund the creation of new goods and services via on-chain referendum decided by all XOR token holders.
```

File: src/tbcd.md [hunk 9]
```diff
@@ -1,3 +1,64 @@
...
+
+## Why Implement Token Bonding Curve Dollars (TBCD)
+
+The SORA ecosystem has grown over the years and now there are many tokens and [eclectic](https://cerestoken.io) [builders](https://adar.com). With growth comes maturity, and SORA has recently been moving towards spinning out XSTUSD, a synthetic stablecoin, into the comprehensive SORA synthetics (XST) platform.
```
Comment (outoftardis) [inline src/tbcd.md]: probably makes sense to link to the xst page

File: src/tbcd.md [hunk 10]
```diff
@@ -1,3 +1,64 @@
...
+
+However, many of the partners we engage with for institutional projects are not ready to price everything in XOR, the native token of the SORA network, yet. Instead, they need a token that is readily convertible to XOR on-demand, yet pegged to the USD.
+
+TBCD (**T**oken **B**onding **C**urve **D**ollars) aims to fill this void by being a token that is convertible to XOR, as a reserve asset of the SORA token bonding curve, while also being worth $1, because the token bonding curve always treats TBCD as having the value of $1 USD.
+
+As you may recall, the [SORA token bonding
+curve](https://wiki.sora.org/token-bonding-curve) is a smart contract,
```
Comment (outoftardis) [inline src/tbcd.md]: this a link to the old wiki, should be a relative link within this doc

File: src/tbcd.md [hunk 11]
```diff
@@ -1,3 +1,64 @@
...
+built right into Polkaswap, which mints XOR in exchange for reserve
+assets, and burns XOR to return the reserve assets. There is a margin
+of 20% between buy and sell, which is used for buying back and burning
+VAL, giving funds for future VAL DAOs, as well as to put into accounts
+that in the future can be used for project funding, and giving
+stipends to SORA citizens.
+
+![The SORA Token Bonding Curve and uses of margin between buy and sell
```
Comment (outoftardis) [inline src/tbcd.md:30]: re ` and uses of margin`: maybe `the uses`? or `and the use of`?

File: src/tbcd.md [hunk 12]
```diff
@@ -1,3 +1,64 @@
...
+functions](/.gitbook/assets/flow-of-funds-new.png)
+The SORA Token Bonding Curve and uses of margin between buy and sell
+functions
```
Comment (outoftardis) [inline src/tbcd.md]:
```suggestion
```

if it's meant to be the description for the image, put it above the image

File: src/changelog.md (hunk 6 above)
Comment (bgodlin) [reply src/changelog.md]: Certainly, I think it's important that we compile an initial list of links

Comment (bgodlin) [reply src/changelog.md]: > Certainly, I think it's important that we compile an initial list of links

What are your thoughts on temporarily removing those links and creating a separate issue specifically for this initiative?

File: src/tbcd.md [hunk 13]
```diff
@@ -1,3 +1,64 @@
...
+means that XOR token holders decide the supply. In this way, TBCD can
+contribute to the productive economy of SORA because builders can be
+funded in TBCD, which can lead to more uses for SORA ecosystem
+tokens.
+XOR is not diluted in purchasing power, because any minting of XOR is done only by giving TBCD to the token bonding curve, so TBCD becomes an asset that balances out any XOR put into circulation; in fact, the token bonding curve over-collateralizes, since there is a 20% margin between buy and sell, and part of that margin is XOR that goes into funds that are not currently distributed and put into circulation.
+
+Because the SORA token is going to target a price point of around
+$600, this means that TBCD will have little utility if XOR costs $600,
+while the secondary market price is so much lower (currently it is
+$3).
```
Comment (outoftardis) [inline src/tbcd.md]: it's a good idea to specify when is "currently": e.g. as of June 2023 or something
also: is it really $3?

File: src/tbcd.md [hunk 14]
```diff
@@ -1,3 +1,64 @@
...
+To make TBCD useful, instead of using the ~$600 price point, it will
+be able to buy XOR from the TBC at the current market price + $1. This
```
Comment (outoftardis) [inline src/tbcd.md:44]:
```suggestion
be possible to buy XOR from the TBC at the current market price plus $1. This
```

File: src/tbcd.md [hunk 15]
```diff
@@ -1,3 +1,64 @@
...
+is still higher than the market price, which will incentivize people
+to avoid dumping TBCD for XOR when the XOR price is low (because they
+won’t get a full $1 of value). For example, if XOR is $3 then TBCD
```
Comment (outoftardis) [inline src/tbcd.md:47]:
```suggestion
won’t get a full $1 of value). For example, if XOR is $3, then TBCD
```

File: src/tbcd.md [hunk 16]
```diff
@@ -1,3 +1,64 @@
...
+XOR for TBCD from the SORA token bonding curve will be 20% less than
+the price for buying.
+
+## Algorithmic Central Banking
+
+TBCD is a great example showcasing the possibilities of the SORA token bonding curve to create monetary innovation for the SORA economy. TBCD solves both the question of how to fund the SORA economy’s productive expansion (the creation of new goods and services) and how to build up reserves in the token bonding curve to create a backing aimed at increasing the stability of XOR. By minting TBCD via on-chain referendum, builders can get the resources they need to buidl up the SORA economy and platform, while any TBCD they sell to the SORA token bonding curve will be owned by the SORA ecosystem as protocol-owned liquidity.
+
+While not operating at the official token bonding curve price, TBCD
+still acts as a form of “training wheels” for the SORA economy, where
+the token bonding curve can be activated and build up some assets,
```
Comment (outoftardis) [inline src/tbcd.md:63]:
```suggestion
the token bonding curve can be activated and used to build up some assets,
```

File: src/tbcd.md [hunk 17]
```diff
@@ -1,3 +1,64 @@
...
+selling XOR via a primary market at a premium to the free-floating
```
Comment (outoftardis) [inline src/tbcd.md:64]:
```suggestion
while selling XOR via a primary market at a premium to the free-floating
```

File: src/changelog.md (hunk 7 above)
Comment (bgodlin) [reply src/changelog.md]: @WRRicht3r do we also need the changelogs for the projects mentioned above?

File: src/changelog.md (hunk 6 above)
Comment (outoftardis) [reply src/changelog.md]: tbh I'd comment this part out for now

Comment (WRRicht3r) [reply src/changelog.md]: I have added the changelog link for it to just circle back here. The SORA team will provide the changelogs for the mobile apps soon, the other changelogs need some more research

File: src/val.md [hunk 18]
```diff
@@ -1,10 +1,19 @@
-# VAL (≚\*\*)
+# VAL (≚)
 
-VAL is used to reward validators on the SORA Network.
+- VAL is a new token used to reward SORA network validators that secure the network, created by [community governance](https://medium.com/sora-xor/sora-v2-implementation-1febd3260b87); all contracts and executable code were released in a disabled form and could not be used until activation by a community member (by insertion of cryptographic proof of the referendum); now it is fully activated
+- VAL fully diluted supply will be 100,000,000. Only 33,900,000 VAL are currently circulating and the supply is capped until SORA v2 network launch
```
Comment (outoftardis) [inline src/val.md]:
```suggestion
- VAL is a new token used to reward SORA network validators that secure the network, created by [community governance](https://medium.com/sora-xor/sora-v2-implementation-1febd3260b87); all contracts and executable code were released in a disabled form and could not be used until activation by a community member (by insertion of cryptographic proof of the referendum); now it is fully activated.
- VAL fully diluted supply will be 100,000,000. Only 33,900,000 VAL are currently circulating and the supply is capped until SORA v2 network launch.
```

File: src/val.md [hunk 19]
```diff
@@ -1,10 +1,19 @@
...
+- The VAL token contract is also available on [Etherscan](https://etherscan.io/token/0xe88f8313e61a97cec1871ee37fbbe2a8bf3ed1e4).
 
-- [Decreasing supply](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8) over time, with tokens burned at every SORA v2 network transaction
+- VAL has [Decreasing supply](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8) over time, with tokens burned at every SORA v2 network transaction
```
Comment (outoftardis) [inline src/val.md]:
```suggestion
- VAL has [Decreasing supply](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8) over time, with tokens burned at every SORA v2 network transaction.
```

File: src/val.md [hunk 20]
```diff
@@ -1,10 +1,19 @@
...
 - Elastic rewards to validators and stake nominators are given in VAL, as a percentage of the daily burned VAL
-- Holders receive a part of the XOR created by the token bonding curve to provide liquidity on Polkaswap. VAL holders can group together into DAOs to pool their liquidity and vote to provide liquidity to trading pairs on Polkaswap and receive PSWAP tokens as a reward
+- Holders receive a part of the XOR created by the token bonding curve
+  to provide liquidity on Polkaswap. VAL holders can group together
+  into DAOs to pool their liquidity and vote to provide liquidity to
+  trading pairs on Polkaswap and receive PSWAP tokens as a reward
```
Comment (outoftardis) [inline src/val.md]:
```suggestion
  trading pairs on Polkaswap and receive PSWAP tokens as a reward.
```

File: src/changelog.md (hunk 7 above)
Comment (WRRicht3r) [reply src/changelog.md]: Would be good to have for transparency's sake. I will request the teams these links

File: src/val.md [hunk 21]
```diff
@@ -1,10 +1,19 @@
...
+
+  VAL is the validator reward token for the SORA Network, used to reward those that stake XOR as part of the Nominated Proof-of-Stake consensus algorithm. [As described](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8), VAL is a deflationary token that gets burned with each transaction on the SORA network. A portion of the burned tokens is then re-minted and distributed to validators and stake nominators, in accordance with the reward rules.
```
Comment (outoftardis) [inline src/val.md]: either
```suggestion
- VAL is the validator reward token for the SORA Network, used to reward those that stake XOR as part of the Nominated Proof-of-Stake consensus algorithm. [As described](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8), VAL is a deflationary token that gets burned with each transaction on the SORA network. A portion of the burned tokens is then re-minted and distributed to validators and stake nominators, in accordance with the reward rules.
```
or
```suggestion
VAL is the validator reward token for the SORA Network, used to reward those that stake XOR as part of the Nominated Proof-of-Stake consensus algorithm. [As described](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8), VAL is a deflationary token that gets burned with each transaction on the SORA network. A portion of the burned tokens is then re-minted and distributed to validators and stake nominators, in accordance with the reward rules.
```

File: src/val.md [hunk 22]
```diff
@@ -1,10 +1,19 @@
...
+
+> [VAL tokens were formerly v1 XOR tokens](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8), but were turned into VAL with new tokenomics, through [community governance](https://medium.com/sora-xor/sora-v2-implementation-1febd3260b87). VAL is a multichain token that lives on the SORA v1 Network and on Ethereum, with a trustless bridge—_HASHI_—that spans the two networks.
```
Comment (outoftardis) [inline src/val.md]: why `>`? the `::: info` is a better markup for this

```suggestion
::: info
[VAL tokens were formerly v1 XOR tokens](https://medium.com/sora-xor/sora-validator-rewards-419320e22df8), but were turned into VAL with new tokenomics, through [community governance](https://medium.com/sora-xor/sora-v2-implementation-1febd3260b87). VAL is a multichain token that lives on the SORA v1 Network and on Ethereum, with a trustless bridge—_HASHI_—that spans the two networks.
:::
```

File: src/changelog.md (hunk 6 above)
Comment (outoftardis) [reply src/changelog.md]: it's just misleading when the link is supposed to take you to the android changelog but brings you to the sora docs instead

File: src/tbcd.md (hunk 13 above)
Comment (WRRicht3r) [reply src/tbcd.md]: removed altogether

File: src/tbcd.md (hunk 11 above)
Comment (WRRicht3r) [reply src/tbcd.md:30]: uses of the margin is the most apropos

**Review by outoftardis — APPROVED** [review]
